src/styles/index.scss

```diff
@@ -1,3 +1,122 @@
-body {
+$font-family: "Roboto", arial, sans-serif;
+$text-color-main: #060b35;
+$text-color-secondary: #616070;
+$button-color: #00acdc;
+$margin-bottom: 16px;
+
+*,
+*::after,
+*::before {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: $font-family;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #ffff;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    /* width: 166px; */
+    margin: 0 0 4px;
+    color: $text-color-main;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $text-color-secondary;
+    margin-bottom: $margin-bottom;
+    font-style: normal;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    align-items: flex-end;
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-color-main;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: $margin-bottom;
+    align-items: center;
+    color: $text-color-main;
+
+    &-word {
+      font-size: 12px;
+      font-weight: 400;
+    }
+
+    &-num {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-size: 14px;
+    text-transform: uppercase;
+    font-weight: 700;
+    text-decoration: none;
+    cursor: pointer;
+    color: white;
+    background-color: $button-color;
+    border: 1px solid $button-color;
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+
+  }
+
+  &__link:hover {
+    background-color: #ffff;
+    color: $button-color;
+  }
+}
+
+.stars {
```

Comment:
according to [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) for each BEM block we should have 1 file - please create 1 file for card and 1 for stars, and move their piece of code